sample-ios-app/build.gradle.kts

```diff
@@ -48,17 +41,5 @@ open class BuildIosSampleTask : DefaultTask() {
 }
 
 if (Target.shouldDefineTarget(project, Target.IOS)) {
-    tasks.register<BuildIosSampleTask>("build") {
-        val sampleMppModuleTasks = project(":sample-mpp-module").tasks
-        releaseFramework.set(
-            sampleMppModuleTasks
-                .named<FatFrameworkTask>("fatIosRelease")
-                .map { it.fatFrameworkDir }
-        )
-        debugFramework.set(
-            sampleMppModuleTasks
-                .named<FatFrameworkTask>("fatIosDebug")
-                .map { it.fatFrameworkDir }
-        )
-    }
+    tasks.register<BuildIosSampleTask>("build")
```

Comment:
Does `gradlew clean :sample-ios-app:build` works? Without dependency or manual invoking of `:sample-mpp-module` it won't compile.

Reply:
Yes, there is a new build phase in the Xcode project, which runs the new Gradle task

Reply:
Here is the build phase: https://github.com/badoo/Reaktive/blob/2e15413a2c5f565fedf582e9367301c49d25e00a/sample-ios-app/sample-ios-app.xcodeproj/project.pbxproj#L176